### ISSUE TITLE
Improve SelectVariants rsID matching

### ISSUE THREAD

**Issue by vdauwera** [broadinstitute/gatk#5260]
### Tool(s) or class(es) involved
SelectVariants (all versions -- tested most recently with 4.0.9.0)

### Description
SelectVariants has an option for excluding specific sites by rsID, `--exclude-ids`. Currently the matching is done by exact string matching. The limitation is that if I have a site with two rsIDs (as appears in the 1000 Genomes dataset), I can't filter against just one of them. For example, if I want to exclude this site from 1000G participant HG00096 (because it makes my synthetic data generator choke, if you must know):

    3523879:16	68401338	rs554836707;rs56090907	CGTGCGCTGCT	CTGCT,C	100	PASS <snip>

I have to specify `--exclude-ids 'rs554836707;rs56090907'` in my SelectVariants command. I would like to be able to specify just `--exclude-ids rs554836707` or `--exclude-ids rs56090907` with the same result (the site is excluded). Right now if I do use just one like that, the site is not excluded because the pattern fails to match.

The reason I want to be able to do that is because there are several other similar sites that I need to exclude, so I make an array of rsIDs to exclude, e.g. provide 

````
Array[String] = [
    "rs575450111",
    "rs151266607",
    "rs557010312",
    "rs564559134",
    "rs560466806",
    "rs140802196",
    "rs554836707",
    "rs56090907"
]
````

as an input to my WDL, which handles it in the command block as 

    --exclude-ids ${sep=' --exclude-ids ' exclusionList}

Right now I have to use 

````
[
"'rs575450111;rs151266607'",
"'rs557010312;rs564559134'",
"'rs560466806;rs140802196'",
"'rs554836707;rs56090907'"
]
````

which is stylistically ugly and would not cover cases where only one rsID is annotated for whatever reason.

**Comment by davidbenjamin** [comment]
@MartonKN It looks like the relevant code is lines 713-5 of `SelectVariants`:
```
if (rsIDsToRemove != null && !rsIDsToRemove.isEmpty()) {
            compositeFilter = compositeFilter.and(new CountingVariantFilter(new VariantIDsVariantFilter(rsIDsToRemove).negate()));
}
```
and in `VariantIDsVariantFilter`
```
public boolean test(final VariantContext vc) {
        return includeIDs.contains(vc.getID());
    }
```
This seems like it should do the right thing, except that `vc.getID()` is a `String`, *not* a `List<String>`.  @vdauwera can we assume that a `VariantContext` ID will not contain a semicolon, and therefore we can parse by splitting on `';'`?

**Comment by davidbenjamin** [comment]
Closed by #5464.
